python/lsst/ts/ataos/ataos_csc.py

```diff
@@ -1383,3 +1530,8 @@ def pneumatics_m2s_callback(self, data):
 
         """
         self.pneumatics_m2_state = ATPneumatics.AirValveState(data.state)
+
+    async def close(self):
+
```

Comment:
Also call `close` on the other remotes...

Reply:
Does the the super().close() do that?
I guess not... but then i suppose i'm confused what it does...
I've added 
        await self.camera.close()
```
await self.camera.close()
await self.atspectrograph.close()
```

Reply:
No, `super().close()` will only close the CSC own stuff... for the remotes you need to call them individually...